spec/spec_helper.rb

```diff
@@ -97,3 +97,12 @@
   Kernel.srand config.seed
 =end
 end
+
+def sign_in(user)
+  visit root_path
+  click_link 'Sign In'
+
+  fill_in 'Email', with: user.email
+  fill_in 'Password', with: user.password
+  click_button 'Sign In'
+end
```

Comment:
Nice - in another PR you may want to go through all your tests and refactor to use this where necessary!